__run_eee.sh

```diff
@@ -186,7 +186,7 @@ while [[ "${finished}" = false ]] ; do
     n_model_runs=$(( ${n_model_runs} + ${nlines} - ${skip} ))      # number of model runs
     echo 'number model runs: '${n_model_runs}
 
-    python ${isdir}/codes/${model_function} -i ${parafile_M} -s ${skip} -o model_output.pkl
+    python3 "${isdir}"/codes/${model_function} -i "${parafile_M}" -s ${skip} -o model_output.pkl
```

Comment:
could you please revert "python3" back to "python". I can show you how to tell your system to use "python" as default. Using "python3" here would be quite restrictive to users.